docs/getting-started.md

```diff
@@ -40,30 +40,67 @@ curl localhost:10101/status
-
-Although Pilosa doesn't keep the data in a tabular format, we still use the terms "columns" and "rows" when describing the data model. We put the primary objects in columns, and the properties of those objects in rows. For example, the Star Trace project will contain an index called "repository" which contains columns representing Github repositories, and rows representing properties like programming languages and tags. We can better organize the rows by grouping them into sets called Fields. So the "repository" index might have a "languages" field as well as a "tags" field. You can learn more about indexes and fields in the [Data Model](../data-model/) section of the documentation.
+In order to better understand Pilosa's capabilities, we will create a sample project called "Star Trace" containing information about 1,000 popular Github repositories which have "go" in their name. The Star Trace index will include data points such as programming language and stargazers—people who have starred a project.
 
-#### Create the Schema
+Although Pilosa doesn't keep the data in a tabular format, we still use the terms "columns" and "rows" when describing the data model. We put the primary objects in columns, and the properties of those objects in rows. For example, the Star Trace project will contain an index called "repository" which contains columns representing Github repositories, and rows representing properties like programming languages and stargazers. We can better organize the rows by grouping them into sets called Fields. So the "repository" index might have a "languages" field as well as a "stargazers" field. You can learn more about indexes and fields in the [Data Model](../data-model/) section of the documentation.
 
-Note:
-If at any time you want to verify the data structure, you can request the schema as follows:
+<div class="note">
+ <p>If at any time you want to verify the data structure, you can request the schema as follows:<\p>
+<\div>
 
 ``` request
 curl localhost:10101/schema
 ```
 ``` response
-{"indexes":null}
+{
+  "indexes": [
+    {
+      "name": "repository",
+      "options": {
+        "keys": false,
+        "trackExistence": true
+      },
+      "fields": [
+        {
+          "name": "language",
+          "options": {
+            "type": "set",
+            "cacheType": "ranked",
+            "cacheSize": 50000,
+            "keys": false
+          }
+        },
+        {
+          "name": "stargazer",
+          "options": {
+            "type": "time",
+            "timeQuantum": "YMDH",
+            "keys": false,
+            "noStandardView": false
+          }
+        }
+      ],
+      "shardWidth": 1048576
+    }
+  ]
+}
 ```
+<div class="note">
+	<p>Note: This is the response you should receive once completing this project. It has also been formatted using `jq`. <\p>
```

Comment:
the backticks (`jq`) don't work inside the <div> note section. `<code>jq</code>` would work. It might also be nice to have that be a link to the program site (https://stedolan.github.io/jq/)